# Message describing the sensor joint stiffnesses
Header header
string[] names
float32[] stiffnesses